doc/_admin-guide/080_Log/020_Buffering/002_How_to_get_information/004_Orphan.md

```diff
@@ -2,19 +2,19 @@
 title: Orphan disk-buffer files
 id: adm-log-diskbuff-orphan
 description: >-
-    This section describes orphan disk-buffer files used in syslog-ng Open
-    Source Edition(syslog-ng OSE).
+    This section describes orphan disk-buffer files used in {{ site.product.short_name }} Open
```

Comment:
Apart from the double long-short usage, this can be simplified to `{{ site.product.name }}({{ site.product.short_name }})`